src/schemas/Program/index.js

```diff
@@ -255,6 +255,19 @@ const resolveSingleProgram = async (egoToken, programShortName) => {
 	return response ? convertGrpcProgramToGql(programDetails) : null;
 };
 
+const resolveHTTPProgram = async (programShortName) => {
+	const response = await programService.getProgramPublicFields(programShortName);
+	return response;
+};
+
+const programServicePrivateFields = [
+	'commitmentDonors',
+	'submittedDonors',
+	'genomicDonors',
+	'membershipType',
+	'users',
+];
+
```

Comment:
Create a variable for the private fields. More fields could be added in the future if needed.